elections.md

```diff
@@ -1,55 +1,36 @@
-# Kubernetes Steering Committee Elections 
+# Kubernetes Steering Committee Elections
 
-This document outlines the process, for steering committee elections.  
+This document outlines the process, for steering committee elections.
 
-**NOTE:** This document is stale and requires reevaluation prior to next election cycle. 
 
-## Elections
-
-Special note: The bootstrap committee pledges to recuse itself from any direct
-election activities while they serve as continuity members. Members of the
-bootstrap committee will refrain from endorsing or otherwise advocating for any
-candidate (with the exception that the members of the bootstrap committee may
-vote in the elections, and may choose to run in the 2019 election).
-
-### Members of Standing
+### Eligibility for voting
 
-Initial standing in the Kubernetes community will be defined by the union of:
-* SIG leads
-* Approvers and reviewers in any Kubernetes owned repositories
-* Anyone with write access to a Kubernetes owned repository
+Eligibility to vote for steering committee members will be defined by:
 
-The bootstrap committee *explicitly* believes that this heuristic will be
-inaccurate and not represent the entire community. An exception form will be
-provided for people who have contributed to the project but may not meet these
-criteria. Exception applications will be approved by the bootstrap committee
-will vote on this eligibility, with simple majority determining eligibility.
-The exception process will be used as data points for refining the criteria for
-the future.
+* People who have at least 50 [contributions to the Kubernetes project over the past year as defined by devstats](https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All) as of 2018-08-15
+* People who have submitted the [voting exception form] and are accepted by
+  the election committee. Acceptance will be defined by a simple majority
+  vote, and the criteria used during this process will be used to help refine
+  further elections.
 
-Within its first year, the Steering Committee will redefine these criteria
-around a more robust idea of community membership.
+It is the responsibility of the steering committee to refine these criteria prior
+to each election.
 
 ### Eligibility for candidacy
 
 Anyone may nominate either themselves or someone else to be a candidate in the
```

Comment:
The nomination process is currently listed as TBD in the 2018 voting guide, does this cover it or are there logistics that need to be defined (ie: last year kubernetes-dev@ was used)

Reply:
Yeah so we can do this two ways, either via mailing list like we did last year (which included things in Tim's email that should be codified), or we could do it via github like some other projects do.